model-optimizer/mo/ops/deformable_convolution.py

```diff
@@ -13,16 +13,18 @@ def __init__(self, graph: Graph, attrs: dict):
         super().__init__(graph, {
             'type': __class__.op,
             'op': __class__.op,
-            'version': 'opset1',
+            'version': 'opset8',
             'infer': Convolution.infer,
             'group': 1,
             'deformable_group': 1,
             'multiplication_transparent': True,
             'multiplication_transparent_ports': [(0, 0), (2, 0)],
             'in_ports_count': 3,
             'out_ports_count': 1,
+            'bilinear_interpolation_pad': False,
         }, attrs)
 
     def backend_attrs(self):
-        # the same attributes as in a regular convolution and one additional attribute 'deformable_group' and 'group'
-        return Convolution(self.graph, {}).backend_attrs() + ['deformable_group', 'group']
+        # the same attributes as in a regular convolution and additional attributes 'deformable_group', 'group'
+        # and 'bilinear_interpolation_pad'
+        return Convolution(self.graph, {}).backend_attrs() + ['deformable_group', 'group', 'bilinear_interpolation_pad']
```

Comment:
Here is the bug introduced. For the "opset1" version of the operation we should not add `bilinear_interpolation_pad` attribute. Implement a solution like in the `model-optimizer/extensions/ops/topk.py`

Reply:
fixed in https://github.com/openvinotoolkit/openvino/pull/6806